src/components/App.js

```diff
@@ -202,10 +203,22 @@ class App extends React.Component {
    */
   render() {
     const { todos, editIndex, tab, search } = this.state;
+
     const btnText = editIndex !== null ? 'Save' : 'Add';
```

Comment:
Constants in different file.

```
import * as appConstant from '../constants';

const btnText = editIndex !== null ? 'appConstant.SAVE' : 'appConstant.ADD';
```

Following code is used in multiple places so we can assign the code in a variable.
```
const hasIndex = editIndex !== null;

const btnText = hasIndex ? 'appConstant.SAVE' : 'appConstant.ADD';
```